src/render/md.jl

```diff
@@ -1,17 +1,43 @@
 ## Docs-specific rendering ––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
 
-function writemime(io::IO, mime::MIME"text/md", docs::Docs{:md})
+function writemime(io::IO, mime::MIME"text/md", docs::Docs{:md}, mdstyle::Dict{ASCIIString, ASCIIString} = DEFAULT_MDSTYLE)
     println(io, docs.data)
 end
 
 ## General markdown rendering ------------------------–––––––––––––––––––––––––––––––––––
 
-function save(file::String, mime::MIME"text/md", doc::Metadata; mathjax = false, include_internal = true)
+const DEFAULT_MDSTYLE = Dict{ASCIIString, ASCIIString}([
+  ("header"         , "#"),
+  ("objname"        , "####"),
+  ("meta"           , "**"),
+  ("exported"       , "##"),
+  ("internal"       , "##"),
+])
```

Comment:
Since the keys are all single words we could use `Symbol`s rather, so `:header`, etc.

Also, `UTF8String` as the value so we can cater to people wanting to use something like `# §` as their `:header` style.